pkg/pquerier/sql_parser_test.go

```diff
@@ -76,3 +76,22 @@ func TestParseQuery(t *testing.T) {
 		})
 	}
 }
+
+func TestNegativeParseQuery(t *testing.T) {
+	testCases := []struct {
+		input string
+	}{
+		{input: "select columnA as something, columnB as something"},
+		{input: "select avg(columnA) as something, columnB as something"},
+		{input: "select avg(*) as something"},
+		{input: "select avg(cpu), max(cpu) as cpu"},
+	}
+	for _, test := range testCases {
+		t.Run(test.input, func(tt *testing.T) {
+			_, _, err := pquerier.ParseQuery(test.input)
+			if err == nil {
```

Comment:
Maybe be more specific?

Reply:
there isn't much more i can output, since there is not error.
I can add something like an "expected error" to the test case struct but, mehh.. I think its clear enough and you'll have to debug it anyway.